### ISSUE TITLE
Font not antialiased

### ISSUE THREAD

**Issue by rofrol** [Eugeny/terminus#1479]
![image](https://user-images.githubusercontent.com/28787/64441723-b335d200-d0ce-11e9-8973-29b14837f359.png)

**Comment by Eugeny** [comment]
Looks antialiased to me... 
![image](https://user-images.githubusercontent.com/161476/64441871-03ad2f80-d0cf-11e9-9fb0-60d3f5cfc81c.png)

**Comment by rofrol** [comment]
It looks much better in Tilix

![image](https://user-images.githubusercontent.com/28787/64441972-43741700-d0cf-11e9-98bd-a83a524f8e12.png)

or Gnome Terminal

![image](https://user-images.githubusercontent.com/28787/64442019-5edf2200-d0cf-11e9-87c0-50f470642c11.png)

**Comment by rorlork** [comment]
I have the same problem - font doesn't have proper sub-pixel antialiasing, only grayscale antialiasing.

Version: 1.0.103
Platform: Windows 10 x64 18363.657
Plugins: none

From electron's issues it seems it might be caused due to not setting the window to an opaque color:
- https://github.com/electron/electron/issues/10025
- https://github.com/electron/electron/issues/6344

That brought me to the Electron FAQ: https://github.com/electron/electron/blob/master/docs/faq.md#the-font-looks-blurry-what-is-this-and-what-can-i-do

This problem doesn't occur to me in other Electron apps like Discord or VSCode, but it does appear on this app:

| Discord | VS Code | Terminus |
|-----|------|------|
| ![image](https://user-images.githubusercontent.com/1844478/75626039-46564d00-5bc4-11ea-9130-da45df976186.png) | ![image](https://user-images.githubusercontent.com/1844478/75626054-6f76dd80-5bc4-11ea-8090-48813bf36062.png) | ![image](https://user-images.githubusercontent.com/1844478/75626075-99300480-5bc4-11ea-9e7c-ac0db9cd4c38.png) |

**Comment by rorlork** [comment]
No, bot, that they haven't looked into the issue more doesn't mean that it has been fixed.

**Comment by rofrol** [comment]
Yea bot, good boy.

**Comment by shaddu38** [comment]
Who have similar problem, you can try old version with "hterm" setting.
Tabby version 1.0.151
Settings->Terminal->Frontend: hterm
There i don't have this issue.

**Comment by jax7778** [comment]
I updated recently, and all of a sudden I had this issue, didn't roll back, but I changed the front end to xterm (canvas) , instead of the default xterm(WebGL) and the problem immediately went away.

**Comment by dezbracaty** [comment]
This problem still exisits....

**Comment by dezbracaty** [comment]
This problem still exisits....

**Comment by XiGou** [comment]
> Who have similar problem, you can try old version with "hterm" setting. Tabby version 1.0.151 Settings->Terminal->Frontend: hterm Restart Tabby There i don't have this issue.

Thans Bro, I changed the setting to canvas instead of webGL, it gets better.

**Comment by gazhay** [comment]
Can confirm GPU acceleration makes text render poorly, disabling makes it work normally.
